lib/interfaces/jwt-module-options.interface.ts

```diff
@@ -10,16 +10,16 @@ export interface JwtModuleOptions {
   signOptions?: jwt.SignOptions;
   secret?: string | Buffer;
   publicKey?: string | Buffer;
-  privateKey?: jwt.Secret;
+  privateKey?: string | Buffer;
```

Comment:
This needs to change since the jwt.Secret typescript def also includes a 3rd option beyond strings and Buffers for overloads that exist in the jwt.verify method.